security.rst

```diff
@@ -715,6 +715,10 @@ URL pattern. You saw this earlier, where anything matching the regular expressio
                 # require ROLE_ADMIN for /admin*
                 - { path: ^/admin, roles: ROLE_ADMIN }
 
+                # the 'path' value can be any valid regular expression
+                # (this one will match URLs like /api/entity1/528491 and /api/entity2/528491)
+                - { path: ^/api/(entity1|entity2)/\d+$, roles: ROLE_USER }
```

Comment:
The Symfony docs are known to use real-life examples. Let's rename `entity1` and `entity2` to real life entities (i.e. `post` and `comment`?)